core/arch/arm/plat-imx/drivers/imx_csu.c

```diff
@@ -80,6 +81,21 @@ const struct csu_config csu_imx6sl = { NULL, csu_setting_imx6sl };
 const struct csu_config csu_imx6sx = { NULL, csu_setting_imx6sx };
 const struct csu_config csu_imx7ds = { NULL, csu_setting_imx7ds };
 
+static void rngb_configure(vaddr_t csu_base)
```

Comment:
Hmmm ... I don't find this very elegant but I guess it's because of the CSU driver not being very fexible ...
A good improvement for the CSU driver would be to have something similar to TZC driver and have functions like:
```
imx_csu_init(vaddr_t csu_base)
imx_csu_configure(unsigned int csu_index, uint16_t permissions)
[...]
````
available to other drivers to lock (or not) registers.

Reply:
who maintains this driver?